src/components/Editor.svelte

```diff
@@ -203,7 +203,7 @@
             <div class="flex flex-col gap-1 mt-2">
                 <h4 class="font-medium">Title: {paraData.title}</h4>
                 <h4 class="font-medium">Author: {paraData.author}</h4>
-                <h4 class="font-medium">Source: {paraData.source}</h4>
+                <h4 class="font-medium">Source: <a href={paraData.resource} target="_blank">{paraData.resource}</a></h4>
```

Comment:
make link blue to make it obvious and give it a underline on hover

Reply:
use these classes `hover:underline text-blue-800`

Reply:
Done!